plugins/scaffolder/src/components/useOwnUser.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { UserEntity } from '@backstage/catalog-model';
+import { identityApiRef, useApi } from '@backstage/core';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+import { useAsync } from 'react-use';
+import { AsyncState } from 'react-use/lib/useAsync';
+
+/**
+ * Get the catalog User entity (if any) that matches the logged-in user.
+ */
+export function useOwnUser(): AsyncState<UserEntity | undefined> {
+  const catalogApi = useApi(catalogApiRef);
+  const identityApi = useApi(identityApiRef);
+
+  // TODO: get the full entity (or at least the full entity name) from the
```

Comment:
I'd remove this. Seems a bit too copy and pasta

Reply:
Forgot that I don't even use this `useOwnUser` any more :D